build_projects/dotnet-host-build/PrepareTargets.cs

```diff
@@ -42,6 +42,17 @@ public class PrepareTargets
         public static BuildTargetResult Init(BuildTargetContext c)
         {
             var configEnv = Environment.GetEnvironmentVariable("CONFIGURATION");
+            string platformEnv = Environment.GetEnvironmentVariable("TARGETPLATFORM") ?? RuntimeEnvironment.OperatingSystemPlatform.ToString();
+            string targetRID = Environment.GetEnvironmentVariable("TARGETRID");
```

Comment:
>We so need a lot of work on core-setup build system.

Yes, we really do. Once CLI finishes the transition to msbuild, core-setup is going to be the black sheep with a nonconforming structure and build system. 

IMO it would be worthwhile to put in the time and effort to bring it up to structural parity with the other repos. It was no easy task transitioning from CoreFX to this, and it's only going to get worse as the build system evolves.